rfc8724.xml

```diff
@@ -90,7 +90,7 @@ Low-Power Wide Area Network (LPWANs) to UDP and IPv6
 the title) for use on https://www.rfc-editor.org/search.
 -->
 
-<keyword>example</keyword>
+<keyword>lpwan</keyword>
```

Comment:
compression and fragmentation in the title, so apparently they don't need to appear again in the list of keywords.
lpwan does not appear in the currently proposed title, it'LPWANs, so just to make sure...

Reply:
I will not limit the use of SCHC to LPWANs; I will not add it to the Title

Reply:
I was leaning taht way too. Let see what others think.